src/cli/commands/refs-local.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+const { print } = require('../utils')
+
+module.exports = {
+  command: 'refs-local',
```

Comment:
@alanshaw it seems like it will be tricky to make the parser work with both
`ipfs refs <path>`
`ipfs refs local`

I can probably hack it to make it work, but before I do that do you think it would make more sense to the user to give the commands different names instead?

Reply:
Lets keep compat with go-ipfs - it's not too hacky to detect when `path === 'local'` and do the right thing in the `refs` handler.

Reply:
That part is easy, it's making the command line parser / help text auto-formatter understand it that will be a little trickier, I'll see if I can make it work

Reply:
Good point, please take a look but if it's going to be a big time sink or over complicated then lets not worry.

Reply:
I added a command aliaser at the beginning of parsing (unfortunately it's not possible to do with yargs middleware): https://github.com/ipfs/js-ipfs/pull/2004/files#diff-71bd1bde77e38a68f76195b8f85ce19c